source/topologies.txt

```diff
@@ -0,0 +1,36 @@
+============================
+Supported Cluster Topologies
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:ref:`c2c-mongosync` supports two cluster topologies: 
+
+- :ref:`Replica sets <replication>`
+
+- :ref:`Sharded clusters <sharded-cluster>`
+
+To sync a replica set to a replica set, see the :ref:`Quick Start Guide
+<c2c-quickstart>`.
+
+To sync a replica set to a sharded cluster, see :ref:`c2c-rs-to-sharded`.
+
+To sync a sharded cluster to a sharded cluster, see :ref:`c2c-sharded-clusters`.
+
+.. note::
+   
+   - If you are syncing sharded clusters, they are not required to have the same 
+   number of shards.
```

Comment:
Aligning all the text in the bullets will fix the staging issue!

```suggestion
   - If you are syncing sharded clusters, they are not required to have the same 
     number of shards.
```

Reply:
Good catch, sorry I missed that. Fixing now!